package/utils/busybox/patches/410-rngseed.patch

```diff
@@ -0,0 +1,66 @@
+--- /dev/null	2019-05-14 12:36:31.663493429 -0700
++++ busybox-1.28.4/coreutils/rngseed.c	2019-05-16 16:58:03.170495363 -0700
```

Comment:
Shouldn't this line be `b/coreutils/rngseed.c`? When quilt applies this patch as it currently is, wouldn't it just end up creating `build_dir/<your_target>/busybox-X.YY.Z/busybox-1.28.4/coreutils/rngseed.c`?

Reply:
This patch was created based on openwrt-18.06 release. I'm happy to update the patch. But I it appears to be applied with `patch -p1` so it doesn't matter which busybox version is actually used.

Reply:
I'm wondering if you have sent this patch upstream, to the Busybox project as well? Or do you plan to do so? I simply see no point to maintain this separately in the OpenWrt tree, it looks like it should be usable by the wider audience and thus should be probably accepted in the upstream project.

Reply:
@ynezz My first step was to see how it was received by the OpenWRT community. If this is a workable solution for OpenWRT, I’ll push it up to BusyBox so we don’t need to maintain a separate patch.